docs/guide/displaying_children_of_a_content_item.md

```diff
@@ -77,6 +78,41 @@ In this case the `controller` key points to the Query Controller's `locationQuer
 
 This template makes use of the `items` specified in `assign_results_to` to list every child of the folder.
 
+## Using the Content query Field Type
+
+You can also use the same Query Type as above to list the children of a Content item using the Content query Field Type.
+
+To do this, add a Content query Field to your Folder Content Type's definition.
+
+Select "Location children" as the Query type. Provide the `parentLocationId` parameter
+that the Query type requires:
+
+```
+parentLocationId: '@=mainLocation.id'
+```
+
+To customize the display template, in your [standard view configuration](../guide/content_rendering.md#configuring-views-the-viewprovider) file,
+under `content_view`, add a section that indicates the matcher and template to use: 
+
+``` yaml
+content_query_field:
+    folder:
+        match:
+            Identifier\ContentType: folder
+        template: full/folder.html.twig
+```
+
+Then, provide the template in `templates/full/folder.html.twig`.
```

Comment:
I wouldn't use `full` as a directory part for the query field, but `content_query` (it also prepares the ground for future improvements).